src/test/scala/chiselTests/util/LTLSpec.scala

```diff
@@ -0,0 +1,350 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests.util
+
+import chisel3._
+import chisel3.testers.BasicTester
+import chisel3.util.ltl._
+import _root_.circt.stage.ChiselStage
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import Sequence._
+
+class LTLSpec extends AnyFlatSpec with Matchers {
```

Comment:
can you add a scaladoc this

Reply:
I don't think we need ScalaDoc on the spec, but definitely on the actual APIs